rust/OpenMod.Rust/OpenMod.Rust.csproj

```diff
@@ -12,8 +12,8 @@
     <PackageReference Include="OpenMod.Rust.Redist" Version="2260.199.1">
       <IncludeAssets>compile</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="OpenMod.UniTask" Version="2019.2.2.5" />
-    <PackageReference Include="OpenMod.UnityEngine.Redist" Version="2019.4.10">
+    <PackageReference Include="OpenMod.UniTask" Version="2021.2.4.1" />
+    <PackageReference Include="OpenMod.UnityEngine.Redist" Version="2021.3.28.1">
```

Comment:
Did you check Unity version of Rust before changing this?

Reply:
yeah, Rust version is 2022 now

Reply:
but I can be wrong, I just saw somewhere that they have now 2022 Unity Engine